app/models/training_module.rb

```diff
@@ -170,4 +170,15 @@ def tab_label
   def tab_anchor
     tab_label.parameterize
   end
+
+  # @return [String]
+  def card_title
+    coming_soon = 'Coming soon - ' if draft?
+    "#{coming_soon}Module #{id}: #{title}"
+  end
+
+  # @return [String]
+  def card_anchor
+    "#{Rails.application.routes.url_helpers.course_overview_path}#module-#{id}-#{title.downcase.parameterize}"
```

Comment:
Can you do the appending of the anchor to `course_overview_path` in the template please